src/_includes/partials/global/footer.njk

```diff
@@ -1,29 +1,53 @@
 <footer role="contentinfo">
-    <div class="container">
-        <div class="social-media">
-            <nav aria-label="{% __ 'social-media' %}">
-                {% __ 'social-media' %}
-                <ul role="list">
-                    <li><a href="https://www.facebook.com/" rel="external">{% include "svg/facebook.svg" %} Facebook</a></li>
-                    <li><a href="https://www.linkedin.com/" rel="external">{% include "svg/linkedin.svg" %} LinkedIn</a></li>
-                    <li><a href="https://twitter.com/" rel="external">{% include "svg/x.svg" %} X</a></li>
-                    <li><a href="https://www.youtube.com/" rel="external">{% include "svg/youtube.svg" %} YouTube</a></li>
-                </ul>
-            </nav>
+    <div class="footer__container">
+        <div class="footer__title">
+            <div><b>inclusive</b></div>
+            <div>standards</div>
         </div>
-        <div class="contact-us">
-            {% __ 'contact-us' %}
-            <div class="contact-us-item">
-                {% __ 'contact-us-email' %}
+        <div class="footer__content">
+            <div class="footer__section">
+                <hr>
+                <div class="footer__social-media">
+                    <nav aria-label="{% __ 'social-media' %}">
+                        <h4>{% __ 'social-media' %}</h4>
+                        <ul role="list">
+                            <li><a href="https://www.facebook.com/" rel="external">{% include "svg/facebook.svg" %} Facebook</a></li>
+                            <li><a href="https://www.linkedin.com/" rel="external">{% include "svg/linkedin.svg" %} LinkedIn</a></li>
+                            <li><a href="https://www.instagram.com/" rel="external">{% include "svg/instagram.svg" %} Instagram</a></li>
+                            <li><a href="https://twitter.com/" rel="external">{% include "svg/x.svg" %} X</a></li>
+                            <li><a href="https://www.youtube.com/" rel="external">{% include "svg/youtube.svg" %} YouTube</a></li>
```

Comment:
You should add the social media links to the site's global data and load them from there rather than hardcoding these generic links.